Source/WebCore/rendering/RenderLayerFilters.cpp

```diff
@@ -169,10 +169,10 @@ GraphicsContext* RenderLayerFilters::beginFilterEffect(RenderElement& renderer,
     if (!m_filter)
         return nullptr;
 
-    auto& filter = *m_filter;
+    Ref<CSSFilter> filter = *m_filter;
```

Comment:
I think for smart pointers we generally don't specify the template parameter unless required.